bazel/cas/service.go

```diff
@@ -8,10 +8,10 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"math/rand"
 	"net"
 	"sync"
 	"time"
-
 	"github.com/golang/protobuf/proto"
 	log "github.com/sirupsen/logrus"
 	remoteexecution "github.com/twitter/scoot/bazel/remoteexecution"
```

Comment:
related to above comment, as long as you're in here, could you move this down to the scoot stuff below? thanks!